doc/reference/kernel/synchronization/events.rst

```diff
@@ -0,0 +1,133 @@
+.. _events:
+
+Events
+######
+
+An :dfn:`event object` is a kernel object that implements a traditional
+events.
+
+.. contents::
+    :local:
+    :depth: 2
+
+Concepts
+********
+
+Any number of event objects can be defined (limited only by available RAM). Each
+event object is referenced by its memory address. One or more threads may wait
+on an event object until the desired set of events has been posted to the event
+object. When new events are posted to the event object, all threads whose
+wait conditions have been satisfied become ready simultaneously.
+
+An event object has the following key properties:
+
+* A 32-bit value that tracks which events have been posted to it.
+
+An event object must be initialized before it can be used.
+
+Events may be **posted** by a thread or an ISR. When posting events, the
+events may either overwrite the existing set of events or add to them in
+a bitwise fashion. Posting events has the potential to fulfill match conditions
+of multiple threads waiting on the event object. All threads whose match
+conditions have been are made active at the same time.
```

Comment:
Sentence doesn't read right. I believe you want "matched" after "been"

Reply:
Good catch. It was 'met' that I wanted, but accidentally missed.